snap/snapcraft.yaml

```diff
@@ -68,15 +67,14 @@ parts:
     build-attributes:
       - enable-patchelf
     prime:
-      - "usr/lib/git-core"
-      - "usr/share/git-core"
-      - "usr/lib/x86_64-linux-gnu/libcurl-gnutls*"
-      - "usr/lib/x86_64-linux-gnu/libpsl*"
-      - "usr/lib/x86_64-linux-gnu/librtmp*"
-      - "usr/lib/x86_64-linux-gnu/libnghttp2*"
-      - "usr/lib/x86_64-linux-gnu/libldap*"
-      - "usr/lib/x86_64-linux-gnu/liblber*"
-      - "usr/lib/x86_64-linux-gnu/libsasl2*"
+      - "-usr/bin"
+      - "-usr/share/doc"
```

Comment:
Chisel releases have a requirement to [include the copyright file](https://github.com/canonical/chisel-releases/blob/95a72dca1e15bbb22fa1266f1a833df712cfc87e/slices/git.yaml#L57-L59) - we might need to do that here too.

Reply:
Is there an official requirement for snaps?

There's not an easy way to do this because the `stage` and `prime` keywords don't allow excluding everything except a particular file.  We would need extglob support to say `-usr/share/doc/*/!(copyright)`.

One alternative is to run `shopt -s extglob; rm -r usr/share/doc/*/!(copyright)` in an override-build script.